packages/sdks/src/types/builder-content.ts

```diff
@@ -12,6 +12,10 @@ export interface BuilderContentVariation {
     jsCode?: string;
     tsCode?: string;
     httpRequests?: { [key: string]: string };
+    breakpoints?: {
```

Comment:
Why are we adding it to content data here? I thought we decided to use meta property for this, since in general `data` is for customer added properties and data?

Reply:
https://builder-internal.slack.com/archives/C01V9EH4T1N/p1663611462024529?thread_ts=1663610512.642029&cid=C01V9EH4T1N

Reply:
Thankyou! I keep confusing myself. Fixed it now.